go.sum

```diff
@@ -4,6 +4,8 @@ github.com/aws/amazon-kinesis-firehose-for-fluent-bit v0.0.0-20190723183349-6ca3
 github.com/aws/amazon-kinesis-firehose-for-fluent-bit v0.0.0-20190723183349-6ca31170fc03/go.mod h1:jZjLd+hsaK0oNVotfqbMK9gUsIhXvYEl7Z3tdZTYa54=
 github.com/aws/aws-sdk-go v1.20.6 h1:kmy4Gvdlyez1fV4kw5RYxZzWKVyuHZHgPWeU/YvRsV4=
 github.com/aws/aws-sdk-go v1.20.6/go.mod h1:KmX6BPdI08NWTb3/sm4ZGu5ShLoqVDhKgpiN924inxo=
```

Comment:
Old SDK is still in the go.sum

Fix this by running `go mod tidy`

Reply:
For some reason, `go mod tidy` does not fix this issue. It looks like, we might need a new release for amazon-kinesis-firehose-for-fluent-bit where this dependency is coming from!

Reply:
So I finally figured this out. By default, go mod uses the latest git tag from a repo- which is `v1.0.0` for firehose. 

But you can tell it to just use the latest master:

```
github.com/aws/amazon-kinesis-firehose-for-fluent-bit@master
```

I ran that on your branch, and it fixed it.